### ISSUE TITLE
No IE plaintext support?

### ISSUE THREAD

**Issue by darth90rs** [drinchev/monosocialiconsfont#4]
Hi guys!
I find this font amazing, very complete and useful.
I also found that in IE (from 9 below) the support for plaintext icon is not working, the font is displaying correctly just if using the html entities.

Ex: 
in chrome/firefox this is working: <span class="symbol">facebook</span>
in IE the above is not working, but the following does: <span class="symbol">&#xe027;</span>

It would be great to have support for social name to display the relative icon also in IE, but I don't know how to, so if someone is fond of this please help!

**Comment by drinchev** [comment]
Yep, that's an issue with IE 9 and less. They all don't support ligature symbol replacement. All similar fonts doesn't work with this. 

I'm going to make a simple JS script to replace the characters automatically for IE, this weekend. Stay tuned!

**Comment by darth90rs** [comment]
Great, that's awesome! I'll be waiting, thank you very much ;)

**Comment by nhrx** [comment]
Hi, could you also make a fallback script for Opera?
Thanks for this font, it's very useful!

**Comment by matthewtlh** [comment]
I'm curious if you got the JS working for IE character replacement.

**Comment by drinchev** [comment]
Thanks for the support guys, I really appreciate it.

@matthewtlh,

Yep, I'm pretty much at the testing phase. I'm on a vacation now, so this will stall a bit ( ETA 2 weeks ). It will support Opera and it is jQuery free. It will support also Ligature Symbols [1] too. Also it can be configured to use any other symbol font that use ligatures substitution. 

Although I don't recommend to use the ligature substitution on AJAX websites ( just use the unicode symbol `&#xeXXX;` ).

Cheers.

[1] https://github.com/kudakurage/LigatureSymbols/issues/3

**Comment by bfintal** [comment]
I also need this :(

**Comment by mikeselander** [comment]
Is the js fix still in the works? Would love to use your awesome font, but the complete lack of IE support means I can't...

**Comment by WesCossick** [comment]
For those still wondering, the fix is quite simple, and does not need JS. Just include this line in your CSS:
`-ms-font-feature-settings:"liga" 1;`

**Comment by mikeselander** [comment]
Oh fantastic, thank you!

**Comment by danwild** [comment]
So glad I used plain text and only realized this in final testing :s haha.
(Still very grateful for the font though thanks!)

**Comment by shoshanna** [comment]
I'm using the rounded style, and for some reason the fix above worked on every icon except "roundedpinterest" and "roundedinstagram".  Is there anything special about those two icons that needs some other fix on IE?

**Comment by WesCossick** [comment]
Looking at @drinchev's icon demo website, a handful of icons experience this issue in IE (and even other browsers like Safari). For those, it seems you'll just need to use the unicode symbols.

**Comment by shoshanna** [comment]
Thank you!

**Comment by invot** [comment]
Where in the file would I put `-ms-font-feature-settings:"liga" 1;`?
It doesn't seem to be working for me. Also, anyone found a JS fix yet?

**Comment by mikeselander** [comment]
It should go in your CSS in the same class that you're using the font with
